akka-actor/src/main/scala/akka/actor/ReflectiveDynamicAccess.scala

```diff
@@ -41,6 +48,15 @@ class ReflectiveDynamicAccess(val classLoader: ClassLoader) extends DynamicAcces
       createInstanceFor(c, args)
     }
 
+  override def classIsOnClasspath(fqcn: String): Boolean = {
+    getClassFor(fqcn) match {
+      case Failure(_: ClassNotFoundException | _: NoClassDefFoundError) =>
+        false
+      case _ =>
+        true
+    }
+  }
```

Comment:
I'd remove the `{} around the function since not all functions already have it, but that's just me.